### ISSUE TITLE
This ISO Builder doesn't work anymore

### ISSUE THREAD

**Issue by manlycracker65** [Humming-Owl/SMG2-Mod-Builder-To-USB-Loader#9]
![image](https://github.com/Humming-Owl/SMG2-Mod-Builder-To-USB-Loader/assets/103338779/d6815efb-5a41-4c5d-9a17-21c464f8f05b)

I have a working ISO and it won't detect it

**Comment by Humming-Owl** [comment]
hmm let me see.
Which is the exact (complete name) of the ISO file?
Maybe its file extension is ".ISO" with big letters. If it is that change it to ".iso"

**Comment by manlycracker65** [comment]
Super Mario Galaxy 2.iso is the name

**Comment by Humming-Owl** [comment]
Try removing the spaces on the name. Put it like "SMG2.iso". If that not solves the issue then try replacing the batch file `drag&drop.bat` with the one I am sending and send a screenshot of the output.
[drag&drop.zip](https://github.com/Humming-Owl/SMG2-Mod-Builder-To-USB-Loader/files/12074351/drag.drop.zip)

**Comment by manlycracker65** [comment]
Tried doing both and still doesn't work

**Comment by manlycracker65** [comment]
![image](https://github.com/Humming-Owl/SMG2-Mod-Builder-To-USB-Loader/assets/103338779/a91ed04e-8b65-47ec-816f-f4552974e030)

**Comment by Humming-Owl** [comment]
are you dragging and dropping the ISO file into the `drag&drop.bat` file? it seems like if you were just double clicking the `drag&drop.bat` file.

**Comment by manlycracker65** [comment]
Ah, you meant to literally drag and drop it, not open it then drag and drop

**Comment by Humming-Owl** [comment]
Yeah, you literally drag and drop the ISO/WBFS into the BAT file directly so the program starts.
Does it work now?

**Comment by manlycracker65** [comment]
Yeah it works now > the

**Comment by Humming-Owl** [comment]
Good, enjoy the hacks!
